<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	/*border-image: url(:/backgrounds/background_temp.jpg) 0 0 0 0 stretch stretch;*/
	color: white;	
	background-color: rgb(0,0,0);
	/*border-left: 2px solid #760b1e;*/
	border-radius:10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <widget class="QPushButton" name="fpc_enabler">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>40</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/point_scan.svg</normaloff>:/icons/point_scan.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="prototype_txt_enabler">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>80</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/texture.svg</normaloff>:/icons/texture.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="camera_preview_enabler">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>120</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/camera.svg</normaloff>:/icons/camera.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="shaders_enabler">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>160</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/shaders.svg</normaloff>:/icons/shaders.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="environment_enabler">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>200</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/environment.svg</normaloff>:/icons/environment.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="create_btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>420</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter Black</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:#760b1e;
	border-radius: 15px;
	color: white;
	border-left: none;
}</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
	color: white;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string>2D</string>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/3D.svg</normaloff>:/icons/3D.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
	color:white;
}
QPushButton:checked{
	background-color: transparent;
	border-bottom: 2px solid white;
	border-bottom-color:red; 
}</string>
     </property>
     <property name="text">
      <string>3D</string>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/icons/3D.svg</normaloff>:/icons/3D.svg</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="project_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro Medium</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 0;
	border-bottom: 2px solid white;
	border-bottom-color: #760b1e;
    outline: 0;
	font-size: 1.3rem;
	color:white;
	/*padding: 7px 0;*/
	background-color: transparent;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Project Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="engine_logo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter Black</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	text-align: left;
	color: white;
	border: none;
}</string>
     </property>
     <property name="text">
      <string> REDOT</string>
     </property>
     <property name="icon">
      <iconset resource="main_resources.qrc">
       <normaloff>:/logo/redot_monochrome.svg</normaloff>:/logo/redot_monochrome.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="phrase_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Cherokee</family>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background: none;
	border-image: none;
	border-left: none;
}</string>
     </property>
     <property name="text">
      <string>Open‑source game engine for everyone.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter Display</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeWidget{
	background: none;
	background-color: rgba(0, 0, 0, 22);
	padding: 10px;
	border-radius:10px;
	border-image: none;
	border-left: none;
}

QScrollBar{
	width:1px;
	margin:0px;
	padding:0px;
	background-color: transparent;
}

QScrollBar::handle{
	background-color: none;
}

QHeaderView{
	border-radius:10px;
	background:  rgba(0,0,0,0);
	border-image: none;
	border-left: none;
}

QHeaderView::section {
    background-color: rgba(0, 0, 0, 22);
    color: white;
    padding-left: 4px;
	border: none;
	border-radius:10px;
	font: 270 10pt &quot;Inter&quot; ;
}

QHeaderView::section:checked
{
    background-color: red;
}

/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <column>
      <property name="text">
       <string>Main Paths</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>autoloads</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>addons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>assets</string>
      </property>
      <item>
       <property name="text">
        <string>models</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>resources</string>
      </property>
      <item>
       <property name="text">
        <string>scenes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>materials</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>scripts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>shaders</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>visual_shaders</string>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>raw_addons</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="main_image">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>401</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: transparent;
	border-left: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="main_resources.qrc">:/lateral_images/redot_chan.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
	color: white;	
}</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minimize_btn">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Inter</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: transparent;
	border: none;
	color: white;	
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <zorder>main_image</zorder>
    <zorder>fpc_enabler</zorder>
    <zorder>prototype_txt_enabler</zorder>
    <zorder>camera_preview_enabler</zorder>
    <zorder>shaders_enabler</zorder>
    <zorder>environment_enabler</zorder>
    <zorder>create_btn</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>project_name</zorder>
    <zorder>engine_logo</zorder>
    <zorder>phrase_label</zorder>
    <zorder>treeWidget</zorder>
    <zorder>exit_btn</zorder>
    <zorder>minimize_btn</zorder>
   </widget>
   <widget class="QPushButton" name="godot_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main_resources.qrc">
      <normaloff>:/logo/godot_normal.svg</normaloff>:/logo/godot_normal.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="redot_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main_resources.qrc">
      <normaloff>:/logo/redot_normal.svg</normaloff>:/logo/redot_normal.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Enter, Esc</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ursina_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main_resources.qrc">
      <normaloff>:/logo/ursina.webp</normaloff>:/logo/ursina.webp</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color:  rgb(22, 22, 22);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>frame</zorder>
   <zorder>godot_btn</zorder>
   <zorder>redot_btn</zorder>
   <zorder>ursina_btn</zorder>
  </widget>
 </widget>
 <resources>
  <include location="main_resources.qrc"/>
 </resources>
 <connections/>
</ui>
